spec/requests/entry_requests_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe "EntryRequests", type: :request do
```

Comment:
```suggestion
describe "EntryRequests" do
```

I note it here that we were talking about, it is great that you covered the authorization with tests, but I'd rather keep it as a temporary step that should not be merged into the master. When the other PR is merged all of these authorization rules will be tested twice. I'd recommend to keep here one for the 'happy path' and later on you can add one for a failed authorization, where you can verify the policy is called before the action.